nasa-gibs/worldview
# WV-2641 end dates

## Description
Layers that are no longer producing new imagery are calculated as unavailable 1 day prior to the end date.

Fixes # wv-2641

Examples:
- LIS_Very_High_Resolution_Lightning_Daily_Climatology_LIS_Mean_Flash_Rate (daily product, 1998 JAN 01 to 2013 DEC 31)
- LIS_Very_High_Resolution_Lightning_Seasonal_Climatology_LIS_Mean_Flash_Rate (Every 3 months, 1998 JAN to 2014 JAN 01)
- LIS_Very_High_Resolution_Lightning_Monthly_Climatology_LIS_Mean_Flash_Rate (Monthly product, 1998 JAN to 2014 JAN 01)
- Radiance (Daily product, 2006 JUN 13 to 2011 OCT 31)

Note that you can check the start/end dates via the layer info window or the getCapabilities endpoint if you want/need to do additional confirmation.

## How To Test

- git checkout wv-2641-end_dates
- npm ci
- npm run build
- Confirm incorrect behavior [in production](https://worldview.earthdata.nasa.gov/)
- Verify correct behavior with this build for the layers listed above

@nasa-gibs/worldview
